modules/visual_script/register_types.cpp

```diff
@@ -43,7 +43,7 @@
 
 VisualScriptLanguage *visual_script_language = nullptr;
 #ifdef TOOLS_ENABLED
-static vs_bind::VisualScriptEditor *vs_editor_singleton = nullptr;
+static VisualScriptCustomNodes *vs_editor_singleton = nullptr;
```

Comment:
Should this be renamed too?

```suggestion
static VisualScriptCustomNodes *vs_custom_nodes_singleton = nullptr;
```

Reply:
Sure, done.